lerna-package-lock.json

```diff
@@ -1966,20 +1966,20 @@
 			}
 		},
 		"@fluid-experimental/task-manager-previous": {
-			"version": "0.59.1000",
-			"resolved": "https://registry.npmjs.org/@fluid-experimental/task-manager/-/task-manager-0.59.1000.tgz",
-			"integrity": "sha512-14U5Fumd4UjBHBXsgclXzRo6+GLUGzp2l1CXN6olKufCuKCRpgLwFMpYUk0FcJmabwxkRg2c8+rg5MBw3TdpXg==",
+			"version": "npm:@fluid-experimental/task-manager@0.59.2001",
+			"resolved": "https://registry.npmjs.org/@fluid-experimental/task-manager/-/task-manager-0.59.2001.tgz",
+			"integrity": "sha512-XD4ubrdNJYX8AtIUQE4A7AHRpsrTV16Sbmoan/Rjf8uz7RtEKmA3xh3+tSZayiRf2mPASn0LbT78F4Vr4PzYdw==",
```

Comment:
@anthony-murphy - This is kind of interesting, something I did updated these.  Is this expected?  I wonder if it should have been proactively updated as part of the 59.2000 minor release?

Reply:
Regular dependencies got updated to 2xxx too, but that's less interesting - I think the "previous" packages updating might be able to affect the type validator results...? Trying to think of any example - maybe adding a new prop in next and adding it as optional in a minor release in main.  Maybe that's what's going on with the couple places where I had to _remove_ forwardCompat broken markup in package.json here?

Reply:
this is expected. the back compat version from next to the previous version is a semver: https://github.com/microsoft/FluidFramework/blob/adb061efa4771cc0ea648c0b37f68b1a9c5177a9/packages/runtime/agent-scheduler/package.json#L67

it would probably be a good practice to bump next versions after main release. that will help keep the breaking in sync between the two.

Reply:
I'll update the release instructions.  Should be to just do a local reinstall, right?